test/unit/create-contentful-api-test.js

```diff
@@ -379,7 +379,7 @@ test('CDA call sync fails', (t) => {
 test('Given json should be parsed correctly as a collection of entries', (t) => {
   const api = createContentfulApi({
     http: {},
-    shouldLinksResolve: true
+    getGlobalOptions: sinon.stub().returns({resolveLinks: true})
```

Comment:
A check is missing here if `getGlobalOptions` was actually called

OR just remove the stub and use a function